nipap-cli/nipap_cli/command.py

```diff
@@ -194,10 +191,10 @@ def parse_cmd(self, tree, inp_cmd = None):
             if self.children is not None:
                 self.key_complete = False
                 match = False
-                for param, content in self.children.items():
+                for param, content in list(self.children.items()):
```

Comment:
Hmm, isn't this a dict? Why would 2to3 put a list around that? Doesn't make sense IMHO and I don't understand how it can work.